obal/data/modules/srpm.py

```diff
@@ -54,6 +61,32 @@ def copy_sources(spec_file, package_dir, sources_dir):
             run_command(["git-annex", "lock", "--force"])
 
 
+def fetch_remote_sources(source_location, source_system, sources_dir):
+    """
+    Copy RPM sources from a remote source like Jenkins to rpmbuild environment
+    """
+    source_system_urls = {
+        'jenkins': '{}/lastSuccessfulBuild/artifact/*zip*/archive.zip',
+    }
+
+    url = source_system_urls[source_system].format(source_location)
+    request = urlopen(url)
```

Comment:
Ansible has `open_url`, `fetch_url` and `fetch_file` in https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/urls.py which could be used instead, but I don't think we buy anything by using them here.